.github/workflows/prChecks.yml

```diff
@@ -0,0 +1,26 @@
+name: PR checks
+run-name: Checking PR created by ${{ github.actor }}
+on: [push]
+jobs:
+  Test-PR:
```

Comment:
nit-pick

with the current configuration this action will be run on any pushes to a branch which means 
* when updating a branch
* when the branch is merged into main (which is also a push)

so the name `Test-PR` is a bit inaccurate here. So lets rename to e.g. _build_.

Reply:
on 2nd sight. you use the name `PR` in a number of locations (file name, name, job id), etc.

As described above this action will _not_ just be run on PR's (which is totally fine). So I would suggest to just fix the naming.